src/components/SecondaryPanes/XHRBreakpoints.css

```diff
@@ -82,11 +82,36 @@
 }
 
 .xhr-input-method {
-  display: none;
+  visibility: hidden;
+  margin-inline-start: 4px;
+  padding-block: 0;
+  padding-inline-start: 2px;
+  padding-inline-end: 14px;
+  border: none;
+  border-radius: 2px;
+  font: inherit;
+  color: inherit;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--toolbarbutton-background);
+}
+
+.xhr-input-container.focused::after {
+  content: "";
+  position: relative;
+  right: 25px;
```

Comment:
Can we test this UI in right-to-left direction?

We may need to add LTR/RTL styles here to handle both directions, something like:

```css
html[dir=ltr] .xhr-input-container.focused::after {
  right: 25px;
}

html[dir=rtl] .xhr-input-container.focused::after {
  left: 25px;
}
```

(Ideally we would use `inset-inline-end: 25px`, but that's not supported in Chrome yet.)

Reply:
![image](https://user-images.githubusercontent.com/9259833/57595792-04a4ca80-74fc-11e9-9253-4efa2f9520e5.png)

I put the rules in and tested for RTL layout.